automatic/sourcetree/sourcetree.nuspec

```diff
@@ -0,0 +1,36 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>sourcetree</id>
+    <title>SourceTree</title>
+    <version></version>
+    <authors>Atlassian</authors>
+    <owners>chocolatey</owners>
+    <projectUrl>http://www.sourcetreeapp.com</projectUrl>
+    <iconUrl>https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/d9984ffd9b6481c92e83fc1ec16f55184633adb6/icons/sourcetree.png</iconUrl>
+    <licenseUrl>https://www.atlassian.com/end-user-agreement/</licenseUrl>
+    <docsUrl>https://confluence.atlassian.com/get-started-with-sourcetree</docsUrl>
+    <bugTrackerUrl>https://support.atlassian.com/help</bugTrackerUrl>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/automatic/sourcetree</packageSourceUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <summary>Harness the power of Git and Hg in a beautifully simple application</summary>
+    <description>
+A free visual Git and Hg client for Mac and Windows
+
+SourceTree simplifies how you interact with your Git and Mercurial repositories so you can focus on coding. Visualize and manage your repositories through SourceTree's simple interface.
+
+## Features
+
+* **Commit with Confidence:** Visualize your work and push with confidence. Stage and discard changes by the file, hunk or line.
+* **Visualize your progress:** Detailed branching diagrams make it easy to keep up with your team's progress.
+* **Git Large File Support:** SourceTree supports Git LFS, allowing teams to track large assets in one single place.
+* **Git-flow out of the box:** Smart branching with SourceTree and Git-flow, keeping repositories clean and development efficient.
+* **Submodules:** Submodules make life easier when managing projects, their dependencies, and other project groupings.
+* **Local commit search:** Search for commits, file changes, and branches right within SourceTree.
+* **Interactive rebase:** Get clean and clear commits with SourceTree's interactive rebase tool.
+* **Remote repository manager:** SourceTree allows you to search for and clone remote repositories within its simple user interface.
+</description>
+    <releaseNotes>https://www.sourcetreeapp.com/update/WindowsReleaseNotes.html</releaseNotes>
+    <tags>git mercurial client admin freeware cross-platform</tags>
+  </metadata>
+</package>
```

Comment:
you're missing the files element. i.e:
```xml
<files>
  <file src="tools\**" target="tools" />
</files>
```
If it isn't added, the `update.ps1` file would be included in the package.
this can be added by whoever merge this PR, but a note for future PR's